platzky/db/json_db.py

```
@@ -57,56 +70,70 @@ def get_post(self, slug: str) -> Post:
-    def get_page(self, slug):
-        list_of_pages = (
-            page for page in self._get_site_content().get("pages") if page["slug"] == slug
-        )
+    def get_page(self, slug: str):
+        """Retrieve a page by its slug.
+
+        TODO: Add test for non-existing page.
+        """
+        pages = self._get_site_content().get("pages")
+        if pages is None:
+            raise ValueError("Pages data is missing")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)